apps/site/assets/css/_fare-card.scss

```diff
@@ -44,6 +44,11 @@ $fare-card-icon-offset: 1.375 * $base-spacing;
   &--linked {
     @extend %button-focus-color;
 
+    p a {  
```

Comment:
this will not work. the homepage banners use this utility and the secondary links are _not_ within `<p>` tags. Why can this not just be any link within the parent selector?

Reply:
This utility is used for things besides just fare cards. Hence it's placement in `utilities`